homeserver.yaml

```diff
@@ -858,7 +858,7 @@ media_store_path: "/data/media_store"
 
 # Enable registration for new users.
 #
-#enable_registration: false
+enable_registration: true
```

Comment:
I thought we wanted to use authentication via OCF accounts instead of using registration?

Reply:
This is just temporary so that people can test out the current configuration without troubles, planning to make another PR that will disable registration once authentication is up